.vscode/settings.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Isn't this a Visual Studio-specific config file. If yes, please take this file out of the git commit list using git rm. It should not be checked into this github codebase.

Reply:
Yes it is, accidentally added it. Will take it out.